src/stream.ts

```diff
@@ -54,11 +59,13 @@ export class InboundStream {
   private readonly rawStream: Stream
   private readonly closeController: AbortController
 
-  constructor(rawStream: Stream) {
+  constructor(rawStream: Stream, opts: InboundStreamOpts = {}) {
     this.rawStream = rawStream
     this.closeController = new AbortController()
 
-    this.source = abortableSource(pipe(this.rawStream, decode()), this.closeController.signal, { returnOnAbort: true })
+    this.source = abortableSource(pipe(this.rawStream, decode(opts)), this.closeController.signal, {
```

Comment:
Should we add some default maxDataLength value?

Reply:
we can default to bellatix 10 mb allthough decode itself defaults to 4mb, what do you say?

Reply:
if it has default limit, than it's fine